# Test -failfast flag

! gno test -failfast ./...

! stdout .+
stderr '--- FAIL: TestFail'
stderr 'someError'
! stderr '.*TestFail2'
! stderr 'secondError'


-- failfast.gno --
package failfast

-- failfast_test.gno --
package failfast

import (
	"testing"
)

func TestFail(t *testing.T) {
	t.Run("TestFail1",func(t *testing.T){
		t.Error("someError")
	})

	t.Run("TestFail2",func(t *testing.T){
		t.Error("secondError")
	})
}

func TestFail2(t *testing.T) {
	t.Error("secondError")
}

